src/services/itemLogin/test/index.test.ts

```diff
@@ -559,9 +567,17 @@ describe('Item Login Tests', () => {
               payload,
             });
 
+            expect(res.statusCode).toBe(StatusCodes.OK);
             const member = res.json();
             expectItemLogin(member, m);
-            expect(res.statusCode).toBe(StatusCodes.OK);
+
+            // last authenticated is updated
+            const guestInDb = await rawItemLoginRepository.findBy({
+              name: payload.username,
+              itemLoginSchema,
+            });
+            expect(guestInDb).toHaveLength(1);
+            expect(guestInDb[0].lastAuthenticatedAt).toBeDefined();
```

Comment:
Should we also check that it is different after we logged in ? So we can ensure it is actually updated ?
If the test relies on the fact that it is undefined before, this assumption is not clear and I think it should be avoided. I  mean we should avoid relying on default values. Any values that we rely on in the test should be explicitly defined.